src/questions/intro-to-defi.yaml

```diff
@@ -17,3 +17,360 @@
       key: C
     - content: Encourage money laundering.
       key: D
+- uuid: 41e017dd-a7d7-4ed3-a9b8-97dc33449afc
+  type: MultipleChoice
+  content: What can be possible use cases for DeFi?
+  hint: The use cases of DeFi
+  explanation: DeFi is used for storing asset on blockchain, for lending and borrowing assets, tokenization and asset exchange
+  answerKeys:
+    - A
+    - B
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: For storing value on blockchain
+      key: A
+    - content: For Lending and borrowing assets
+      key: B
+    - content: For exchange of assets
+      key: C
+    - content: For Securing money in vaults
+      key: D
+- uuid: d8ca36fe-1b4d-4390-b05a-419d854a6100
+  type: SingleChoice
+  content: Which of these is an accurate description associated with TradFi
+  hint: Shortcomings of Traditional finance
+  explanation: Traditional finance is handled by a central body, lacks transparency, is inefficient and lacks innovation.
+  answerKeys:
+    - A
+  difficultyLevel: "Low"
+  choices:
+    - content: Controlled by an authority
+      key: A
+    - content: Open source
+      key: B
+    - content: Transparent system
+      key: C
+    - content: Fairly innovative
+      key: D
+- uuid: d091e037-c787-453d-82c8-6dcb1c5b01e2
+  type: MultipleChoice
+  content: What are some features of DeFi?
+  hint: Features of DeFi
+  explanation: DeFi ledger is immutable, transparent and highly efficient 
+  answerKeys:
+    - B
+    - C
+  subTopics:
+    - "introduction-to-decentralised-finance"
+  difficultyLevel: "Low"
+  choices:
+    - content: Limited Access
+      key: A
+    - content: Immutable
+      key: B
+    - content: Highly efficient
+      key: C
+    - content: Opaque
+      key: D      
+- uuid: e989accc-4817-4c06-8611-995d7d069114
+  type: SingleChoice
+  content: What do you understand by interoperability of DeFi?	
+  hint: Features of Decentralised Finance
+  explanation: Interoperability of DeFi refers to interconnection between various dapps in the DeFi ecosystem.
+  answerKeys:
+    - C
+  subTopics:
+    - "uses-of-decentralised-finance"
+  difficultyLevel: "Medium"
+  choices:
+    - content: DeFi is accessible from anywhere.
+      key: A
+    - content: DeFi is free to use for everyone.
+      key: B
+    - content: DeFi applications are interconnected.
```

Comment:
`DeFi applications are interconnected.` this is contextual as most of the apps on ethereum are not interconnected